Updating crit itemgroups.

thelonestander opened the pull request:
Crit has a lot of cool gear, lots of it didn't spawn naturally. This fixes that.

Clothes and armor I have added to army personal lockers.

Added guns to the military distribution as well.


#### Summary Mods "Added itemspawns for the weapons and the clothes and armor."


#### Purpose of change

Crit had a lot of good gear, and cool weapons. So, I added spawns for them so they can actually be found in game. 
For balancing I made the armor spawns quite low, so you'll still need to search for the items. Same with the guns. 

#### Describe the solution

Just added itemgroups for the items and guns. 

#### Describe alternatives you've considered

none

#### Testing

Tested in game looted a few locations and found crit gear and weapons!

#### Additional context

![image](https://user-images.githubusercontent.com/82045140/188962506-9e70edd6-6da0-4705-bd82-9723729fc277.png)

Coolthulhu commented:
> For balancing I made the armor spawns quite low, so you'll still need to search for the items. Same with the guns.

That's not much of a balancing. I don't know how good AFS balance is, so I'd like some context here.

thelonestander commented:
> > For balancing I made the armor spawns quite low, so you'll still need to search for the items. Same with the guns.
> 
> That's not much of a balancing. I don't know how good AFS balance is, so I'd like some context here.

Im going to go through this again here at some point, so I can better define the spawns.

thelonestander commented:
Further fixed up itemgroup spawns. I added some of the armors as a chance to spawn at military surpluses. The dress clothes and base military clothes some armors also spawn in military personal lockers. I moved the some of the guns to spawn in military locations and the energy guns should also spawn with vanilla energy guns, but with a lower chance. Since Crit gear is supposed to be "the best of the best military gear" I made spawns lower as I would assume most who grabbed the gear is still using it or have died since.
